tuhi-gui.in

```diff
@@ -17,16 +17,23 @@ from gi.repository import Gio
 
 
 @devel@  # NOQA
-resource = Gio.resource_load(os.fspath(Path('@pkgdatadir@', 'tuhi.gresource')))
+if "TUHI_DEVEL" in os.environ:
+    resource = Gio.resource_load(os.fspath(Path(os.environ["TUHI_DEVEL"], 'data', 'tuhi.gresource')))
+else:
+    resource = Gio.resource_load(os.fspath(Path('@pkgdatadir@', 'tuhi.gresource')))
```

Comment:
Can we split this up into the path composition and the actual `Gio.resource_load` path please? I'd like to keep the conditional bits as minimal as possible, so something like:

```suggestion
if "TUHI_DEVEL" in os.environ:
    resource_path = Path(os.environ["TUHI_DEVEL"], 'data', 'tuhi.gresource')
else:
    resource_path = Path('@pkgdatadir@', 'tuhi.gresource')
    
 resource = resource = Gio.resource_load(os.fspath(resource_path))
```
Maybe there's even a nicer option using `os.environ.get()` but I couldn't find one that reads easily.